app/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -98,20 +98,20 @@ class EmailVerificationConnectorImpl @Inject() (http: WSHttp, metrics: Metrics)(
 
     status match {
       case BAD_REQUEST ⇒
-        logger.warn(s"[EmailVerification] - received status 400 (Bad Request) (round-trip time: ${nanosToPrettyString(time)})", nino)
+        logger.warn(s"Received status 400 (Bad Request) (round-trip time: ${nanosToPrettyString(time)})", nino)
         Left(RequestNotValidError)
 
       case CONFLICT ⇒
-        logger.info("[EmailVerification] - email address already verified, received status 409 (Conflict) " +
+        logger.info("Email address already verified, received status 409 (Conflict) " +
           s"(round-trip time: ${nanosToPrettyString(time)})", nino)
         Left(AlreadyVerified)
 
       case SERVICE_UNAVAILABLE ⇒
-        logger.warn(s"[EmailVerification] - received status 504 (Service Unavailable) (round-trip time: ${nanosToPrettyString(time)})", nino)
+        logger.warn(s"Received status 504 (Service Unavailable) (round-trip time: ${nanosToPrettyString(time)})", nino)
```

Comment:
Do you mean "Received status 503" (rather than 504)?

Reply:
yes, I've corrected it now